src/stactools/noaa_cdr/sea_ice_concentration/constants.py

```diff
@@ -67,16 +67,23 @@
         "https://nsidc.org/data/g02202/versions/4",
     )
 ]
-LICENSE_LINK = Link(
-    "license",
-    (
-        "https://www.ncei.noaa.gov/pub/data/sds/cdr/CDRs/"
-        "Sea_Ice_Concentration/UseAgreement_01B-11.pdf"
-    ),
-    MediaType.PDF,
-    "NOAA CDR Sea Ice Concentration Use Agreement",
-)
-KEYWORDS = COMMON_KEYWORDS + ["Sea ice", "Polar"]
+# This link is invalid
+# LICENSE_LINK = Link(
+#     "license",
+#     (
+#         "https://www.ncei.noaa.gov/pub/data/sds/cdr/CDRs/"
+#         "Sea_Ice_Concentration/UseAgreement_01B-11.pdf"
+#     ),
+#     MediaType.PDF,
+#     "NOAA CDR Sea Ice Concentration Use Agreement",
+# )
```

Comment:
No need to keep the code around as a comment, we'll trust version control to let us roll it back if we need to :-).

```suggestion
```